heudiconv/cli/run.py

```diff
@@ -99,6 +99,16 @@ def process_extra_commands(outdir, args):
 
 
 def main(argv=None):
+    import etelemetry
+    try:
+        latest = etelemetry.get_project("nipy/heudiconv")
+    except RuntimeError as e:
+        print("Could not check for version updates: ", e)
```

Comment:
is there a >90% guarantee that `etelemetry` wouldn't crash for some other (IO/Syntax/whatever) reason? ;-)  I would have caught all other exceptions (eg. `except Exception as e`) and issued a warning but kept going